src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Command to add a task to the task list.
+ *
+ * @author Cheong Yee Ming
+ * @version Duke Level-9
+ */
+public class AddCommand extends Command {
+    private final Task task;
+
+    /**
+     * Constructor for an AddCommand.
+     *
+     * @param taskList Handles all operations regarding tasks.
```

Comment:
I like how the descriptions have the same indentation which makes it easy to read. 👍